Assets/Scripts/Scriptable Objects/ChargeBehaviour.cs

```diff
@@ -0,0 +1,86 @@
+using UnityEngine;
+using UnityEngine.AI;
+
+[CreateAssetMenu(fileName = "Charge Behaviour", menuName = "Behaviours/Charge")]
+public class ChargeBehaviour : AIBehaviour
+{
+    public int damage = 10; // Damage per collision
+    public float chargeDelay = 2f; // Seconds per charge
+    public float turnRange = 3f; // This is one of the parameters used to cancel a charge early; if the player has gone too far away from the furniture it will stop and turn.
+    public float horizontalKnockbackMultiplier = 0.1f; // Determines how much the horizontal knockback is multiplied. The multiplier should be small to prevent excessive flinging.
+    public float verticalKnockback = 4f; // Determines how hard the player is flung upwards. Not nearly as sensitive and varied as the horizontal one.
+    [Range(0, 180)] public float maxTurnAngle = 120; // Determines the angle in which the furniture decides to stop charging when out of range. Has to be within 0 - 180 degrees since that's the limit for Vector3.Angle
```

Comment:
You could expose this as a 0-360 degree angle and create a private property that is half of the exposed angle for use internally (see `ViewConeSO` if you need an example)